usefulUtilities/modelJSONCleanup/README.md

```diff
@@ -0,0 +1,18 @@
+# Description
+
+Node script that can be used to help cleanup models.json
+
+# Use
+
+- Get an archive of the domain you would like
+- Extract the zip
+- Extract the models gzip file
+- Duplicate the generic js filter file for the specific domain
+- In the JS file add the path to the models.json and the new models.json
+- Follow the comment instructions to tweak to your needs
+- Save and run node "nameOfTheJSFile"
```

Comment:
```suggestion
- Save the modified file and run `node <nameOfTheJSFile>`
```

Reply:
will i need to run `npm install` to get any node packages?

Reply:
Nope, just uses fs and path which come with

Reply:
Maybe I can delete this part all together by moving to the config file implementation.